tests/cmsController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once('../cmsController.php');
+require_once('../portfolioLogic.php');
+
+class StackTest extends TestCase
+{
+    //makeDropDown
+    //success
+
+    public function testDropDownSuccess ()
+    {
+        $input = [['title' => 'test']];
+        $expected = '<option value="test">test</option>';
+        $case = makeDropDown($input);
+        $this->assertEquals($case, $expected);
+    }
+
+    //failure
+    public function testDropDownFailure ()
```

Comment:
isnt this test just the success test copied? :/

Reply:
Removed